argo-install.sh

```diff
@@ -1,11 +1,27 @@
 #!/bin/bash
 
+# Install Argo workflow pods and client
+
 set -euxo pipefail
 
+# Install argo workflow
 kubectl apply -f https://raw.githubusercontent.com/argoproj/argo/stable/manifests/quick-start-postgres.yaml
 kubectl patch configmaps/workflow-controller-configmap -p '{"data":{"containerRuntimeExecutor":"k8sapi"}}'
 
 NS=$(kubectl config view --minify --output 'jsonpath={..namespace}')
 NS=$([ ! -z "$NS" ] && echo "$NS" || echo "default")
 
 kubectl create rolebinding --clusterrole=edit --serviceaccount="$NS":default argo
+
```

Comment:
hmm -- I have regularly run this script on my Mac to install argo into a fresh k8s deploy; all the kubectl stuff works as intended for that.  But the local client install below won't work, and assumes linux machine, assumes sudo, and assumes I don't already have a client installed...  Should argo-client-install.sh be a separate script?